### ISSUE TITLE
Document current externa web page embed function(s)

### ISSUE THREAD

**Issue by tsobuskerudbyen** [buskerudbyen/cycling-norway#81]
Any restrictions or things we should avoid on embedding the website on other pages? 

1. Should we explain something to people that want to use the maps embedded on their web page? 
2. Beware of overuse? 
3. Errors? 
4. Other methods possible than the `<embed> `HTML-tag? 
5. Etc.

Please add some comments here, and I will document it to our known users.

**Comment by leonardehrenfried** [comment]
> 1. Should we explain something to people that want to use the maps embedded on their web page?

I think a documentation page is a good idea. If we decided that we want it, we can also make an official widget which would enable us to get rid of the iframe. We talked about this before so we should know what we are getting ourselves into it when we decide to do it, because there will most likely be 

> 2. Beware of overuse?

At the moment this is not a concern at all, as the tiles served very efficiently. If we area at a point where this matters I think we are established enough to get more funding.

> 3. Errors?

There are probably some, but I can't think of what could go wrong. The most common one is probably domains being blocked.

> 4. Other methods possible than the `<embed> `HTML-tag? 

See above.
